app/views/posts/show.html.erb

```diff
@@ -2,12 +2,17 @@
   <div class="col-lg-8 col-md-11 col-11 border p-lg-2 p-1">
     <div class="d-flex justify-content-between">
       <h5><%= @post.title %> by <%=  @post.user.name %></h5>
+      
       <p>Comments: <%= @post.comments_counter %> , Likes:  <%= @post.likes_counter %></p>
     </div>
     
     <br>
     <p><%= @post.text %></p>
+    <% if can?(:destroy, @post) %>
+    <%= button_to 'Delete Post', user_post_path(@post.user, @post), method: :delete, class: 'btn btn-danger' %>
+  <% end %>
   </div>
+
 </div>
 <div class="row mx-lg-4 mx-md-3 mx-lg-2 mx-1 justify-content-center align-items-center ">
   <div class="col-lg-8 col-md-11 col-11 border p-lg-2 p-1">
```

Comment:
**OPTIONAL** - You have done well so far, but your code throws the error below when trying to add a new post. Please fix this and ensure users can add new posts using the already implemented forms. 
![rgawrgawe](https://github.com/maths29/blog_app/assets/73607512/d55af9ba-731c-46f9-83c6-a61d27d7fcda)

Reply:
Hi @maths29, It would be nice to fix this error as I cannot create a new post 👍🏿 
![Screenshot_230](https://github.com/maths29/blog_app/assets/69638013/8e0f617d-f3ee-4247-a923-78146ea01ddb)